testing/cucumber/features/test_case_scanning_payload.feature

```diff
@@ -11,6 +11,7 @@ Feature: Scan results return useful information
       Scenario:
         * a step
     """
+    And a tag prefix of "@test_case_"
```

Comment:
The feature files are ostensibly user friendly documentation of the gem. As such, any details that aren't relevant to the behavior/usage being demonstrated are best left out of the Gherkin level of the "test". Because `@test_case` is the default prefix used, it is not worth mentioning in this case. Also, it shouldn't have to be added at all because it is the default that will be applied if nothing is passed to the library code. So these new failures would cause me to be suspicious of the implementation, not the test.

Reply:
Yep, there was a bug. `master` has the fix and CI is green again.